lib/manager/poetry/artifacts.spec.ts

```diff
@@ -70,7 +70,9 @@ describe('.updateArtifacts()', () => {
     expect(execSnapshots).toMatchSnapshot();
   });
   it('returns updated poetry.lock', async () => {
-    fs.readFile.mockResolvedValueOnce('Old poetry.lock' as any);
+    fs.readFile.mockResolvedValueOnce(
+      '[metadata]\npython-versions = ">3.6"' as never
```

Comment:
Here's a real example I found:

```
[metadata]
content-hash = "70609fddc0d3768b1003fc24207951ab7ad8bfad4c6cb326d6217c52f5a92e3d"
python-versions = "~2.7 || ^3.4"

```

Reply:
mine is from https://github.com/renovate-tests/poetry1/blob/4f41e4287d81d9ba3688f8a9cf7d3f01bb08d25b/poetry.lock#L126

I've simply removed the hash, as it's not required for tests